internal/provider/datasource_rediscloud_data_persistence_test.go

```diff
@@ -13,8 +13,17 @@ func TestAccDataSourceRedisCloudDataPersistence(t *testing.T) {
 			{
 				Config: testAccDataSourceRedisCloudDataPersistence,
 				Check: resource.ComposeTestCheckFunc(
-					resource.TestCheckResourceAttr(
-						"data.rediscloud_data_persistence.foo", "data_persistence.#", "6"),
+					testCheckResourceValuesInListAnyOrder("data.rediscloud_data_persistence.foo", "data_persistence", []map[string]interface{}{
```

Comment:
Could this be replaced with `resource.TestMatchTypeSetElemNestedAttrs` or `resource.TestCheckTypeSetElemNestedAttrs`?

Reply:
👍  I actually missed those ones, I'll try to use those